examples/window/Window.cpp

```diff
@@ -182,6 +182,4 @@ int main()
         // Finally, display the rendered frame on screen
         window.display();
     }
-
-    return EXIT_SUCCESS;
```

Comment:
I believe this has to stay because it causes the iOS build to fail. For those builds the main function is converted into `sfmlMain` which because it's just a normal function at that point loses this special property where we can omit the return value. I think the same goes for two other examples but CI will eventually catch those errors.